frontend/src/views/InGame.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <h1>Hello {{ player.nickname }}</h1>
+    <h1>Your GameId is {{ game.id }}</h1>
+    <GameState />
+    <Deck />
+  </div>
+</template>
+
+<script lang="ts">
+// @ is an alias to /src
+import Vue from 'vue'
+import io from 'socket.io-client'
+import GameState from '@/components/InGame/GameState.vue'
+import Deck from '@/components/InGame/Deck.vue'
+
+export default Vue.extend({
+  name: 'InGame',
+  components: {
+    GameState,
+    Deck,
+  },
+  computed: {
+    game() {
+      return this.$store.state.game
+    },
+    player() {
+      return this.$store.state.player
+    },
+  },
+  mounted() {
+    if (localStorage.token) {
+      // const websocket = io.connect(process.env.VUE_APP_BACKEND_URL)
+      const websocket = io.connect(process.env.VUE_APP_BACKEND_URL, { autoConnect: false })
```

Comment:
Here, we need the same logic as in https://github.com/martinzuern/cards-offending-mankind/pull/14/files#diff-86e7e8d66da8191a525c19347e6ff535R3